master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -22,12 +22,16 @@ For Kubeadm <=1.6.0-alpha with Kubernetes 1.5.x:
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/calico.yaml
 ```
 
+>[Click here to view the above yaml directly.](calico.yaml)
```

Comment:
I was trying to make it clear that the link was referencing the above install and not the one below. Felt that `>` helped. What is your suggestion? I don't understand what you mean by "inlined".